### PR TITLE
Display DestinationCapacityWarning

### PR DESCRIPTION
**Pull request by kev1n-peters**

### CODE REVIEW

**Comment by kev1n-peters** [comment]
image

**Comment by kev1n-peters** [comment]
image

File: wormhole-connect/src/utils/index.ts [hunk 1]
```diff
@@ -462,3 +462,15 @@ export const millisToRelativeTime = (ts: number): string => {
     return '~1 minute ago';
   }
 };
+
+export const formatDuration = (seconds: number) => {
```
Comment (emreboga) [inline wormhole-connect/src/utils/index.ts:466]: Thought bubble: The number of time formatting utilities is growing :) I think we just need dayjs:
https://day.js.org/docs/en/display/format

**Review by emreboga — APPROVED** [review]
LGTM

**Review by artursapek — APPROVED** [review]

File: wormhole-connect/src/utils/index.ts (hunk 1 above)
Comment (kev1n-peters) [reply wormhole-connect/src/utils/index.ts:466]: Good idea, I'll open an issue for it since the scope is wider than this targeted PR

**Pull request merged**